{{cookiecutter.project_name}}/.gitlab-ci.yml

```diff
@@ -1,48 +0,0 @@
-#
-# GitLab CI Configuration
-#
```

Comment:
I think its still useful to have a basic CI config at least for running tests, if only as an example?

Reply:
👍

Reply:
@jamesjwarren restored a simple gitlab config.